deployment/ansible/pfb.yml

```diff
@@ -8,7 +8,7 @@
 
   roles:
     - { role: "azavea.terraform" }
-    - { role: "pfb.aws-cli" }
+    - { role: "pfb.aws-cli2" }
```

Comment:
Since this PR replaces the `pfb.aws-cli` role, it should include a commit to remove it (probably nice to have that as a separate commit so it would be easy to get it back if we wanted).